apps/stripe/src/app/api/saleor/transaction-initialize-session/use-case-response.ts

```diff
@@ -0,0 +1,70 @@
+import { buildSyncWebhookResponsePayload } from "@saleor/app-sdk/handlers/shared";
+
+import {
+  ChargeFailureResponse,
+  ChargeSuccessResponse,
+} from "@/modules/saleor/saleor-webhook-responses";
+
+import { TransactionInitalizeSessionResponseDataShapeType } from "./response-data-shape";
+
+// TODO: add support for other results e.g AUTHORIZE
+
+class ChargeRequest extends ChargeSuccessResponse {
```

Comment:
I dont get it, why ChargeRequest extends from ChargeSuccess? It will inherit actions like "REFUND" which is not correct yet